static/js/TopicLandingPage/TopicLandingSearch.jsx

```diff
@@ -22,22 +29,31 @@ const getSuggestions = async (input) => {
     }
 
     const _parseSuggestions = (completionObjs, lang) => {
-      let topics = [];
-      if (completionObjs.length > 0) {
-        topics = completionObjs.map((e) => ({
-          title: `#${_capitalizeFirstLetter(e.title)}`,
-          categoryPathTitle: `${_getFormattedPath(e.key, lang)}`,
+      if (completionObjs.length === 0) return [];
+
+      return completionObjs.map((e) => {
+        const categories = Sefaria.topicTocCategories(e.key);
+        const isDisambiguationEqualCategory =
+            categories &&
+            categories.some(category => _extractDisambiguation(e.title) === category[lang]);
+
+        const title = isDisambiguationEqualCategory
```

Comment:
you always want to run `_capitalizeFirstLetter()` in either case. to make this line simpler, move `_capitalizeFirstLetter()` to the return statement.